support.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta name="author" content="" />
+        <!--
+               __                            _
+          ____/ /___  ____ ____  _________  (_)___
+         / __  / __ \/ __ `/ _ \/ ___/ __ \/ / __ \
+        / /_/ / /_/ / /_/ /  __/ /__/ /_/ / / / / /
+        \__,_/\____/\__, /\___/\___/\____/_/_/ /_/
+                   /____/
+
+        so markup. very website. wow.
+        made with love by edswf1
+    -->
+    <title>Dogecoin| Support Dogecoin</title>
+    <!-- Favicon-->
+    <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
+    <!-- preload webfonts -->
+    <link rel="preload" href="assets/vendor/fa/webfonts/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin="anonymous">
+    <link rel="preload" href="assets/vendor/fa/webfonts/fa-brands-400.woff2" as="font" type="font/woff2" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/vendor/fa/css/all.min.css"/>
+    <!-- Google fonts-->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet" type="text/css" />
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&display=swap" rel="stylesheet" type="text/css" />
+    <!-- Core theme CSS (includes Bootstrap)-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css" ></script>
+
+    <link href="assets/css/styles.css" rel="stylesheet" />
+    <style>
+        .support-content{
+            min-height: 300px;
+            max-width: 500px;
+            border: 1px solid #4a6c7d;
+            margin: 0px auto;
+            padding: 20px;
+            border-radius: .5rem;
+        }
+        .support-content .fa{
+            font-size: 2rem;
+        }
+        .support-content .heading{
+            font-size: 1.2rem;
+        }
```

Comment:
i inlined these for my first draft -> i think they should be moved to styles.css

Reply:
seconded.